packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -36,6 +40,8 @@ const CompleteResetPassword = ({
   }, [hasConfirmedRecoveryKey]);
 
   const [isSubmitting, setIsSubmitting] = useState(false);
+  const hasSyncDevices =
+    estimatedSyncDeviceCount !== undefined && estimatedSyncDeviceCount > 0;
```

Comment:
Wondering if we should also consider this true if the service is sync. I went through the flow to reset the password when signing into sync for a second time, and did not see the warning.

Reply:
I was also wondering about this. Happy to change it, although without another conneted device, I'm not sure the first point in the info box will be applicable.

Reply:
Might be applicable if they previously only had one device connected, but are going through the reset process on a new device?